docs/apache-airflow/templates-ref.rst

```diff
@@ -89,16 +89,11 @@ existing code to use other variables instead.
 Deprecated Variable                     Description
 =====================================   ====================================
 ``{{ execution_date }}``                the execution date (logical date), same as ``dag_run.logical_date``
-``{{ next_execution_date }}``           the next execution date (if available) (`pendulum.DateTime`_)
-                                        if ``{{ execution_date }}`` is ``2018-01-01 00:00:00`` and
-                                        ``schedule_interval`` is ``@weekly``, ``{{ next_execution_date }}``
-                                        will be ``2018-01-08 00:00:00``
```

Comment:
I don't think the intent was to remove the example?

Reply:
it was intentional.  firstly, these are all *deprecated*, so folks reading the docs should not be using them anyway.  secondly, by referencing "logical date"  in the definition, we defer to the example in the section of non-deprecated items.

note there is also no example for the also-deprecated value `execution_date`

the example and explanation were confusing anyway so this to me  seem better